### PR TITLE
Code that ignores duplicate points for plotting is removing non-duplicate points

### PR DESCRIPTION
**Pull request by nikhilmandlik**

### CODE REVIEW

File: src/plugins/plot/src/configuration/PlotSeries.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ define([
                     var newPoints = _(this.data)
                         .concat(points)
                         .sortBy(this.getXVal)
-                        .uniq(true, this.getXVal)
+                        .uniq(true, point => [this.getXVal(point), point.eu].join())
```
Comment (akhenry) [inline src/plugins/plot/src/configuration/PlotSeries.js]: `point.eu` is VISTA specific. It's also not the only attribute that users might plot. `point.getYVal()` is a more generic way to retrieve the Y value corresponding to a given X for the currently selected Y axis.

Comment (nikhilmandlik) [reply src/plugins/plot/src/configuration/PlotSeries.js]: ahh yes good catch, thanks

Comment (nikhilmandlik) [reply src/plugins/plot/src/configuration/PlotSeries.js]: fixed in https://github.com/nasa/openmct/pull/2515/commits/0eff81991308869d4ec14b86fcd837c82cba111c

**Comment by akhenry** [comment]
@nikhilmandlik Looks great, thanks!

**Comment by akhenry** [comment]
@nikhilmandlik Would you mind updating this branch from topic-core-refactor? Then I can merge it.

**Comment by deeptailor** [comment]
@nikhilmandlik - Please pull from topic-core-refactor and ill review and merge

**Comment by akhenry** [comment]
## Reviewer Checklist

* Changes appear to address issue? Y
* Appropriate unit tests included? N/A - Plots do not have unit tests
* Code style and in-line documentation are appropriate? Y
* Commit messages meet standards? Y

**Comment by nikhilmandlik** [comment]
**Testathon:** Duplicate point is when both x and y values are same, but old logic was looking into only duplicate of x value.

**Comment by nikhilmandlik** [comment]
**Testathon:**

1. Add following code to test duplicate points.
2. path : openmct/src/plugins/plot/src/configuration/PlotSeries.js
3. After line 161 add following and test Sine wave.

`
points = [
                        {
                            'cos': 0,
                            'mctLimitState': {cssClass: "is-limit--upr is-limit--red", high: Infinity, name: "Red High"},
                            'name': "Sine Wave Generator 1",
                            'sin': 1,
                            'utc': 1575670910000,
                            'yesterday': 1575584547000,
                        },
                        {
                            'cos': -1,
                            'mctLimitState': {cssClass: "is-limit--upr is-limit--red", high: Infinity, name: "Red High"},
                            'name': "Sine Wave Generator 1",
                            'sin': 2,
                            'utc': 1575670910000,
                            'yesterday': 1575584547000,
                        },
                        {
                            'cos': -2,
                            'mctLimitState': {cssClass: "is-limit--upr is-limit--red", high: Infinity, name: "Red High"},
                            'name': "Sine Wave Generator 1",
                            'sin': 3,
                            'utc': 1575670930000,
                            'yesterday': 1575584547000,
                        },
                        {
                            'cos': -3,
                            'mctLimitState': {cssClass: "is-limit--upr is-limit--red", high: Infinity, name: "Red High"},
                            'name': "Sine Wave Generator 1",
                            'sin': 4,
                            'utc': 1575670940000,
                            'yesterday': 1575584547000,
                        },
                        {
                            'cos': -2,
                            'mctLimitState': {cssClass: "is-limit--upr is-limit--red", high: Infinity, name: "Red High"},
                            'name': "Sine Wave Generator 1",
                            'sin': 4,
                            'utc': 1575670950000,
                            'yesterday': 1575584547000,
                        }
                    ];
`
